### PR TITLE
Dataset notifications ui

### PR DESCRIPTION
**Pull request by paragateoslo**

### CODE REVIEW

File: src/config/field-overrides/programNotificationTemplate.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ export default new Map([
     }],
     ['recipientUserGroup', {
         component: (props) => {
+            console.log(props);
```
Comment (varl) [inline src/config/field-overrides/programNotificationTemplate.js]: 👆 Is this a development log line? Either remove it before merging to master or replace it with the log module used for logging elsewhere in the app.

Comment (paragateoslo) [reply src/config/field-overrides/programNotificationTemplate.js]: Ah good catch, that was just for temporary debugging

**Comment by varl** [comment]
LGTM. 👍

**Comment by Birkbjo** [comment]
Looks good. Need to fix conflict though.

**Pull request merged**